aaa.sb

```diff
@@ -0,0 +1,3 @@
+While ("True")
+  TextWindow.WriteLine("a")
```

Comment:
Use `Write` instead of `WriteLine` so it prints on one line

Reply:
clearly you don't know how to make actual suggestions
```suggestion
  TextWindow.Write("a")
```

Reply:
also you can commit to the branch yourself and make the change, especially if it's as small as this

Reply:
> clearly you don't know how to make actual suggestions

i didnt realize that was a thing lmfao

Reply:
<img width="556" alt="Screen Shot 2021-03-19 at 10 56 32 PM" src="https://user-images.githubusercontent.com/32407067/111858373-64453b80-8906-11eb-8549-606736849a60.png">
now you know